src/domains/nodo-app/00_alert_wisp_dismantling.tf

```diff
@@ -34,6 +34,53 @@ AzureDiagnostics
   }
 }
 
+// Query explanation: https://pagopa.atlassian.net/wiki/spaces/I/pages/574751186/Razionalizzazione+Alert
+resource "azurerm_monitor_scheduled_query_rules_alert" "opex_pagopa-wisp-converter-redirect-availability" {
+  count               = var.env_short == "p" ? 1 : 0
+  resource_group_name = "dashboards"
+  name                = "pagopa-${var.env_short}-opex_pagopa-wisp-converter-redirect-availability"
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id, data.azurerm_monitor_action_group.opsgenie[0].id]
+    email_subject          = "Alert pagopa-wisp-converter-redirect-availability"
+    custom_webhook_payload = "{}"
+  }
+
+  data_source_id = data.azurerm_api_management.apim.id
+  description    = "Availability for https://api.platform.pagopa.it/wisp-converter/redirect/api/v1/payments is less than or equal to threshold - https://portal.azure.com/?l=en.en-us#@pagopait.onmicrosoft.com/dashboard/arm/subscriptions/b9fc9419-6097-45fe-9f74-ba0641c91912/resourcegroups/dashboards/providers/microsoft.portal/dashboards/pagopa-p-opex_pagopa-wisp-converter"
+  enabled        = true
+  query = (<<-QUERY
+let lowTrafficThreshold = 80; // the lower threshold that can be calculated regarding the number of invocations
```

Comment:
to select right value
```
let lowTrafficThreshold = 80; // the lower threshold that can be calculated regarding the number of invocations
let highTrafficThreshold = 99; // the upper threshold that can be calculated regarding the number of invocations
let trafficMin = 100; // the minimum number of invocations (traffic) below which 'lowTrafficThreshold' guideline is used
let trafficLinear = 500; 
```
to set the correct values ​​we should perhaps base ourselves on the last week where we also have a real alert

Reply:
Executed a tuning on threshold values as you said.  
As the chart below shows, I have set highest threshold to `95%` in order to reduce noise on physiological errors (i.e. Checkout client error) and the lowest threshold to 70% in order to grant better alerting on low-traffic hours and also track strange behaviors (like the one on the previous week).  

![Screenshot 2024-11-29 alle 10 10 35](https://github.com/user-attachments/assets/de3b0dea-e4e6-4b8f-81c6-702bd28e5667)